readthedocs/analytics/utils.py

```diff
@@ -57,9 +57,6 @@ def anonymize_user_agent(user_agent):
 
 def send_to_analytics(data):
     """Sends data to Google Analytics"""
-    if data.get('uip') and data.get('ua'):
-        data['uid'] = generate_client_id(data['uip'], data['ua'])
```

Comment:
Removing this makes nothing show up in Google Analytics which is even worse. It looks like we want the Client ID (`cid`).